zk_toolbox/crates/zk_inception/src/commands/ecosystem/args/build.rs

```diff
@@ -0,0 +1,102 @@
+use std::{path::PathBuf, str::FromStr};
+
+use clap::Parser;
+use common::{forge::ForgeScriptArgs, Prompt};
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    commands::chain::args::genesis::GenesisArgs,
+    messages::{
+        MSG_ECOSYSTEM_CONTRACTS_PATH_INVALID_ERR, MSG_ECOSYSTEM_CONTRACTS_PATH_PROMPT,
+        MSG_GENESIS_ARGS_HELP,
+    },
+};
+
+const DEFAULT_OUT_DIR: &str = "transactions";
+
+#[derive(Debug, Clone, Serialize, Deserialize, Parser)]
+pub struct EcosystemArgs {
```

Comment:
I think it's conflicting name

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2787/commits/9f8cfd82d14c0620b9c674d63534a072848464e5